packages/ruby/lib/readme/filter.rb

```diff
@@ -11,13 +11,26 @@ def self.for(reject: nil, allow_only: nil)
     end
   end
 
+  def self.redact(rejected_params)
+    rejected_params.each_with_object({}) do |(k, v), hash|
+      # If it's a string then return the length of the redacted field
+      hash[k.to_str] = "[REDACTED#{v.instance_of?(String) ? " #{v.length}" : ""}]"
```

Comment:
`kind_of?` might be nicer to allow subclassing? :shrug:

Reply:
What's the value in providing the length?

Reply:
Including `[REDACTED]` in the first place lets the user know that the field was present in the original request, and not omitted entirely. With the length, the front-end can show those fields like like `****` and `****************************` to better represent the shape of the redacted data.

Reply:
sure happy to change to `kind_of?` for that edge case

Reply:
FYI
<img width="722" alt="image" src="https://user-images.githubusercontent.com/4505008/123856959-b4617000-d8ef-11eb-857b-39847f633176.png">